.github/workflows/validate-source-maps-beta.yml

```diff
@@ -0,0 +1,30 @@
+name: Validate source maps beta
+
+on:
+  workflow_call:
+
+jobs:
+  validate-source-maps-beta:
+    name: Validate source maps beta
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: metamask/github-tools/.github/actions/setup-environment@main
+
+      - name: Download 'dist-beta'
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-beta
+          path: dist-beta
+
+      - name: Download 'builds-beta'
+        uses: actions/download-artifact@v4
+        with:
+          name: builds-beta
```

Comment:
I see that the current job does ensure the `builds` directory is in place before we validate sourcemaps, but.... we don't actually use this directory at all for sourcemap validation. It just uses `dist`.

Reply:
We'll still want to upload the `builds` directory as an artifact, because that's linked in the metamaskbot comment (which we can at some point update to use this artifact rather than the CircleCI link). But we can skip downloading it here.